### PR TITLE
fix: [ANDROAPP-2813]Fix labels for dataSets dataelement column

### PR DESCRIPTION
**Pull request by Balcan**
[issue](https://jira.dhis2.org/browse/ANDROAPP-2813)

### CODE REVIEW

File: app/src/main/java/org/dhis2/usescases/datasets/dataSetTable/dataSetSection/DataSetTableAdapter.java [hunk 1]
```diff
@@ -187,7 +209,7 @@ public void onBindCellViewHolder(AbstractViewHolder holder, Object cellItemModel
 
         rows.get(holder.getItemViewType()).onBind(holder, viewModels.get(rowPosition).get(columnPosition).withValue(cellItemModel.toString()), cellItemModel.toString());
         holder.itemView.getLayoutParams().width = currentWidth;
-        holder.itemView.getLayoutParams().height = currentHeight;
+        holder.itemView.getLayoutParams().height = /*currentHeight*/getColumnHeaderHeight();
```
Comment (JaimeToca) [inline app/src/main/java/org/dhis2/usescases/datasets/dataSetTable/dataSetSection/DataSetTableAdapter.java]: comment?

Comment (Balcan) [reply app/src/main/java/org/dhis2/usescases/datasets/dataSetTable/dataSetSection/DataSetTableAdapter.java]: should be removed yes

File: app/src/main/res/layout/table_view_corner_layout.xml [hunk 2]
```diff
@@ -1,16 +1,45 @@
...
         android:id="@+id/buttonScale"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_centerInParent="true"
-        app:srcCompat="@drawable/ic_zoomx2" />
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toStartOf="@+id/buttonRowScaleAdd"
+        app:layout_constraintStart_toEndOf="@id/buttonRowScaleMinus"
+        app:layout_constraintTop_toTopOf="parent"
+        app:srcCompat="@drawable/ic_zoomx2" />-->
```
Comment (ferdyrod) [inline app/src/main/res/layout/table_view_corner_layout.xml]: Are we going to need this in the near future??

Comment (Balcan) [reply app/src/main/res/layout/table_view_corner_layout.xml]: We are talking about this right now. It seems nope we won't.
I'll update this layout once the icons are decided.

File: app/src/main/java/org/dhis2/usescases/datasets/dataSetTable/dataSetSection/DataSetSectionFragment.java [hunk 3]
```diff
@@ -197,28 +234,45 @@ private void loadHeader(int position) {
...
+            cornerView.findViewById(R.id.buttonRowScaleMinus).setOnClickListener(view -> {
+                        for (int i = 0; i < binding.tableLayout.getChildCount(); i++) {
+                            if (binding.tableLayout.getChildAt(i) instanceof TableView) {
+                                TableView table = (TableView) binding.tableLayout.getChildAt(i);
+                                DataSetTableAdapter adapter = (DataSetTableAdapter) table.getAdapter();
+                                adapter.scaleRowWidth(false);
+                                ViewGroup.LayoutParams params = cornerView.getLayoutParams();
+                                params.width = adapter.getRowHeaderWidth();
+                                cornerView.setLayoutParams(params);
+                                if (i == 0) {
```
Comment (JaimeToca) [inline app/src/main/java/org/dhis2/usescases/datasets/dataSetTable/dataSetSection/DataSetSectionFragment.java]: what does represent i==0?

Comment (JaimeToca) [reply app/src/main/java/org/dhis2/usescases/datasets/dataSetTable/dataSetSection/DataSetSectionFragment.java]: is 0 the header?

Comment (Balcan) [reply app/src/main/java/org/dhis2/usescases/datasets/dataSetTable/dataSetSection/DataSetSectionFragment.java]: A way to save the result just once in the loop

Comment (JaimeToca) [reply app/src/main/java/org/dhis2/usescases/datasets/dataSetTable/dataSetSection/DataSetSectionFragment.java]: why don't you save outside the loop?

**Pull request merged**